test/README.md

```diff
@@ -46,35 +53,34 @@ Run the tests for endpoints in User.ts file
 yarn test:user
 ```
 
-You can run a test in `watch` mode, from the command line. This is helpful while you work on a test, or work on the functionality being tested.
-```sh
-mocha -w someFilePath/someTestFileName
-```
+These tests run under Mocha in watch mode. If you are not familiar with Mocha, take a moment and familiarize yourself with `only` and `skip`. `only` and `skip` are your friends, and can help focus on specific tests as they are being developed, or problems with tests, or when a test fails, etc.
```

Comment:
For the future: I've found it useful to have a linting rule that prevents accidentally commiting `skip` and `only` caveats, so that you can use them during your test writing, but so that you won't accidentally silence a bunch of tests when you commit or push.